cmd/commands/account/command.go

```diff
@@ -103,6 +103,15 @@ func NewCommand() *cli.Command {
 					return nil
 				},
 			},
+			{
+				Name:      "set-identity",
+				Usage:     "Sets a new identity for your account which will be used in commands that require it",
```

Comment:
Does everyone else has a clear understanding of what is an `account` ?

Reply:
My explanation for this would be that: commands are grouped under `account` namespace, while it's not a true user account per say but it's sort of similar. So I think that we can refer to your personal configuration as `account` here.

Though then again it's not to later to rename things, change help messages or event drop the `account` namespace and use something different. I'm very open to ideas.